css/style-for-validation.css

```
@@ -2,8 +2,12 @@
   color: #ff0000;
 }
 
+.ad-form__element:nth-child(8) {
```

```suggestion
.ad-form__element:nth-child(8) {
```
так) ну это сложный селектор) лучше добавь отдельный класс на это